### PR TITLE
Change package manager to yarn

### PR DESCRIPTION
**Pull request by adesege**
This PR changes the package manager to yarn by generating lock files using yarn and changing npm commands to yarn.

Relates to https://github.com/cloudflare/cloudflare-docs/pull/2709

### CODE REVIEW

File: bin/commands.sh [hunk 1]
```diff
@@ -34,8 +34,8 @@ if [ "$1" = "ghactionsbootstrap" ]; then
   echo "Entering .docs"
   cd .docs
 
-  echo "Running npm install inside .docs"
-  npm install
+  echo "Running yarn install inside .docs"
+  yarn install
```
Comment (jrf0110) [inline bin/commands.sh:38]: Since this is the `ghactionsbootstrap` command, I think we should run `yarn install --pure-lockfile` here

Comment (adesege) [reply bin/commands.sh]: Thanks for the feedback. The yarn install command has been updated to reflect your feedback.

**Comment by gganch** [comment]
why the changes in workers-site/package-lock.json ?

**Comment by adesege** [comment]
@gganch The changes are from reverting from yarn install to npm install 'cos workers use npm to install dependencies. I will revert it

**Review by jrf0110 — APPROVED** [review]

**Pull request merged**